x/group/internal/orm/sequence_property_test.go

```diff
@@ -1,3 +1,4 @@
+//nolint:unused // this file contains tests
```

Comment:
What is unused here? Cannot it be deleted if unused?

Reply:
maybe.